src/routes/frontpage/Frontpage.css

```diff
@@ -363,11 +348,13 @@
   cursor: pointer;
 }
 
+/*
 .pressIcon {
   width: 53px;
   height: 43px;
   margin-bottom: 14px;
 }
+*/
```

Comment:
I'm with you. You can always go back in git history to find this code if it needs reintroduction